elfpy/markets/hyperdrive.py

```diff
@@ -599,7 +740,7 @@ def add_liquidity(
             time_remaining=self.position_duration,
         )
         # perform the trade
-        lp_out, d_base_reserves, d_bond_reserves = self.pricing_model.calc_lp_out_given_tokens_in(
+        lp_out, d_base_reserves, d_bond_reserves = self.calc_lp_out_given_tokens_in(
```

Comment:
can you also remove this from the pricing model so we don't have it in two places?